dataplane/README.md

```diff
@@ -0,0 +1,155 @@
+# The Lemming Dataplane
+
+## Overview
```

Comment:
maybe one line about what is a dataplane?

Reply:
done